tr/Economics and Ethics of Private Property Studies in Political Economy and Philosophy/03_p01_ch03.md

```diff
@@ -1,20 +1,20 @@
+Herkesin ekonomik çıkarına göre yalnızca bir evrensel para ve yalnızca bir hesap birimi varken, servetini en üst düzeye ulaştırmak için arayışta olan adam bunu başarıncaya kadar durmaz,  tek bir banka ya da tekelci bir bankacılık sistemi olduğu için bu çıkarlara aykırıdır. Bunun yerine, şahsi çıkarlar her bankanın aynı evrensel parayı kullanmasını emreder ki bu da altın olur; bu durumda farklı paralar arasında rekabet olmaz, ancak hepsi altın kullanan bankalar ile bankacılık sistemleri arasındaki serbest rekabet var söz konusu olur. Yalnızca bankacılığa ücretsiz girme olduğu sürece başka herhangi bir işte olduğu gibi bu konuda maliyet etkinliği olacaktır; ancak bu rekabet bir ve aynı para birimi açısından sunulan hizmetleri ilgilendiriyorsa, bankacılık aslında paranın ve bankacılığın işlevini tam olarak yerine getirebilir, yani parçalanmaktan ziyade ekonomik bütünleşmeyi kolaylaştırıp piyasayı genişletebilir,  işbölümünü sınırlamaktan ziyade işbölümünü genişletebilir, değer ve maliyet muhasebesini daha akılcı yapabilir  ve dolayısıyla ekonomik serveti azaltmaktan ziyade arttırabilir. The notion of competition between monies is a *contradiction in adjecto*. Strictly speaking, a monetary system with rival monies of freely fluctuating exchange rates is still a system of (partial) barter, riddled with the problem of requiring double coincidences of wants in order for (some) exchanges to take place. Böyle bir sistemin varlığı, paranın bir çok amacına aykırıdır.[^4] Şahsi çıkarlarının peşinde özgürce koşan adam hemen terk ederdi -ve paranın özü ile ilgili olarak, serbest piyasa sadece rakip bankalar açısından değil, aynı zamanda rekabet eden para birimleri açısından düşünüldüğünde de temel bir yanlış anlaşılma olacaktır.[^5] Rekabetçi paralar, serbest piyasa hareketlerinin sonucu değil, her zaman, rasyonel ekonomik davranış yolunda hükümet tarafından uygulanan engellerin zorlanması sonucudur.
 
 With free banking based on a universal gold standard emerging, the goal of achieving the most cost efficient solution to coordinating and facilitating interspatial and intertemporal exchanges within the framework of a universally integrated market is accomplished. Prices for the service of safekeeping, transporting and clearing money, as well as for advancing money in time-contracts would drop to their lowest possible levels under a regime of free entry. And since these prices would be expressed in terms of one universal money, they would truly reflect the minimum costs of providing market-integrative services.
```

Comment:
Evrensel bir altın standardına dayalı serbest bankacılığın ortaya çıkmasıyla, evrensel olarak entegre bir pazar çerçevesinde mekan ve zaman dilimleri arası alışverişleri koordine etmek ve kolaylaştırmak için en uygun maliyetli çözümü elde etme hedefi gerçekleştirildi. Tasarruf, taşıma ve para temizleme hizmetlerinin fiyatları ve zamanında para ilerletme sözleşmeleri, serbest giriş rejimi altında mümkün olan en düşük seviyelere inecektir; bu fiyatlar bir evrensel para birimi cinsinden ifade edileceğinden, pazar bütünleştirici hizmet sunmanın asgari maliyetlerini gerçekten yansıtacaktı.